pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -537,9 +537,17 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	}
 	domainCtx.cpuPinningSupported = caps.CPUPinning
 
-	resources, err := hyper.GetHostCPUMem()
-	if err != nil {
-		log.Fatal(err)
+	var resources types.HostMemory
+	for {
+		// Need to wait for things to get started
+		delay := 10
+		resources, err = hyper.GetHostCPUMem()
+		if err == nil {
+			break
+		}
+		log.Warnf("Retrying in %d seconds due to %s", delay, err)
+		time.Sleep(time.Duration(delay) * time.Second)
+		ps.StillRunning(agentName, warningTime, errorTime)
```

Comment:
Should we limit the number of attempts to something meaningful? I think a fatal with a proper log is better than just being stuck forever.

Reply:
Good point (I was originally not putting StillRunning in there so it would watchdog after 5 minutes, but with StillRunning in place it makes sense to have a limit.)

Reply:
Fixed